Dockerfile

```diff
@@ -142,6 +145,9 @@ RUN apk add --no-cache --virtual .pythonmakedepends \
     seqdiag \
   && apk del -r --no-cache .pythonmakedepends
 
+# Installing Nodejs dependencies for additional functionality
+RUN npm install --global @asciidoctor/core asciidoctor-pdf
```

Comment:
Could you fix the NPM package version using the variable `asciidoctor_web_pdf_version` that you introduced + add it to updatecli tracking along with nodejs?

Reply:
OK. let me fix it.